apps/website/src/pages/events/fall-carnival-2023/index.tsx

```diff
@@ -0,0 +1,193 @@
+import { useCallback, useEffect } from "react";
+import { type NextPage } from "next";
+import Image from "next/image";
+import { useSession } from "next-auth/react";
+
+import { env } from "@/env/index.mjs";
+
+import leafLeftImage1 from "@/assets/floral/leaf-left-1-fall.png";
+import leafRightImage2 from "@/assets/floral/leaf-right-2-fall.png";
+import leafLeftImage2 from "@/assets/floral/leaf-left-2.png";
+import IconDownload from "@/icons/IconDownload";
+
+import {
+  eventId,
+  stickerPack,
+  ticketConfig,
+} from "@/data/events/fall-carnival-2023";
+
+import { trpc } from "@/utils/trpc";
+import { getVirtualTicketImageUrl } from "@/utils/virtual-tickets";
+
+import { LoginWithTwitchButton } from "@/components/shared/LoginWithTwitchButton";
+import { ShareButton } from "@/components/shared/ShareButton";
+
+import Section from "@/components/content/Section";
+import Meta from "@/components/content/Meta";
+
+import {
+  TicketEditor,
+  useStickerData,
+} from "@/components/events/virtual-ticket/TicketEditor";
+import { FallCarnival2023Ticket } from "@/components/events/fall-carnival-2023/VirtualTicket";
+import { MovableSticker } from "@/components/events/virtual-ticket/elements/MovableSticker";
+
+import { IntroSection } from "@/components/events/fall-carnival-2023/IntroSection";
+import { Activities } from "@/components/events/fall-carnival-2023/Activities";
+
+const FallCarnival2023EventPage: NextPage = () => {
+  const session = useSession();
+
+  const myTicket = trpc.virtualTickets.getMyTicket.useQuery(
+    { eventId },
+    {
+      enabled: !!session.data?.user,
+      refetchOnMount: false,
+      refetchOnWindowFocus: false,
+      refetchOnReconnect: false,
+    },
+  );
+
+  const saveMutation = trpc.virtualTickets.save.useMutation();
+
+  const stickerData = useStickerData(ticketConfig, stickerPack);
+  const {
+    isDirty,
+    stickerPositionsRef,
+    setStickerData,
+    updateStickerPosition,
+    saveStickers,
+  } = stickerData;
+
+  useEffect(() => {
+    if (myTicket.data) setStickerData(myTicket.data.customization.stickers);
+  }, [myTicket.data, setStickerData]);
```

Comment:
using useEffect feels wrong but we somehow need to sync the data from the server=trpc to the live positions updated by framer motion. there probably is a way to directly pass the trpc/react-query data to the sticker components and update the data from there but i could not figure it out in time